app/locale/en.coffee

```diff
@@ -2111,6 +2113,7 @@ module.exports = nativeDescription: "English", englishDescription: "English", tr
     try_ozaria_footer: "Try our new adventure game, Ozaria!"
     try_ozaria_free: "Try Ozaria for free"
     ozaria_intro: "Introducing Our New Computer Science Program"
+    owned_license: "Owned License"
```

Comment:
Looking at the screenshot, this might be better as "Licensed" or "Has License".

(Also, the screenshot has this text overlapping the CS 1, 2, 3...?